src/ServiceControlInstaller.Engine/Validation/ServiceControlQueueNameValidator.cs

```diff
@@ -21,7 +21,7 @@ public static void Validate(ServiceControlNewInstance instance)
         {
             var validator = new ServiceControlQueueNameValidator(instance)
             {
-                Instances = InstanceFinder.ServiceControlInstances().Where(p => p.Name != instance.Name & p.TransportPackage.Equals(instance.TransportPackage, StringComparison.OrdinalIgnoreCase)).AsEnumerable<IServiceControlTransportConfig>().ToList()
+                Instances = InstanceFinder.ServiceControlInstances().Where(p => p.Name != instance.Name & p.TransportPackage.TypeName.Equals(instance.TransportPackage.TypeName, StringComparison.OrdinalIgnoreCase)).AsEnumerable<IServiceControlTransportConfig>().ToList()
```

Comment:
Again if TransportInfo would have a meaningful Equals method we could probably use that